module/VuFind/src/VuFind/OAI/Server.php

```diff
@@ -996,11 +997,10 @@ protected function listRecordsGetDeleted($from, $until, $currentCursor)
      */
     protected function listRecordsGetDeletedCount($from, $until)
     {
-        $tracker = $this->tableManager->get('ChangeTracker');
-        return $tracker->retrieveDeletedCount(
+        return $this->tracker->retrieveDeletedCount(
             $this->core,
-            date('Y-m-d H:i:s', $from),
-            date('Y-m-d H:i:s', $until)
+            \DateTime::createFromFormat('U', $from, new \DateTimeZone('UTC')),
+            \DateTime::createFromFormat('U', $until, new \DateTimeZone('UTC')),
```

Comment:
Looks like there's an unnecessary extra comma on the end of this line -- I'm surprised that didn't cause a syntax error, but it looks like PHP 8 has become more tolerant. Still probably worth cleaning up for style reasons.